### ISSUE TITLE
Multiple screens on pupils pc

### ISSUE THREAD

**Issue by Taldrit** [iTALC/italc#51]
Hi there,
is it possible to extend italc in the next version, so you can change the screens on the clients side?
We have multiple pcs with 2, 3 oder even more screens, but it seems, that it is only possible to see the first screen. And if that screen has a resolution far from the normal 16:9/4:3, then italc needs massive amounts of time to show it. I.e.: When the client has a vertical resolution (9:16 / 10:16 / 3:4 / etc.).

**Comment by vlknkc** [comment]
@Taldrit Yes, this problem should be resolved. Full screen display resolution is a problem because it is different.

**Comment by FriedmanFr** [comment]
I also would like to have dual monitor capability. Would you like to work together to see if we could get it programmed. email me at ffriedman-at-tampabay-dot-rr-dot-com if you would like to work together on this.
